src/wp-includes/blocks.php

```diff
@@ -1325,7 +1325,10 @@ function build_comment_query_vars_from_block( $block ) {
 			} elseif ( 'oldest' === $default_page ) {
 				$comment_args['paged'] = 1;
 			} elseif ( 'newest' === $default_page ) {
-				$comment_args['paged'] = (int) ( new WP_Comment_Query( $comment_args ) )->max_num_pages;
+				$max_num_pages         = (int) ( new WP_Comment_Query( $comment_args ) )->max_num_pages;
```

Comment:
```suggestion
				$max_num_pages  = (int) ( new WP_Comment_Query( $comment_args ) )->max_num_pages;
```
Doh, fix the alignment.